src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -3262,4 +3262,18 @@ GenTree* Compiler::gtNewSimdAllTrueMaskNode(CorInfoType simdBaseJitType, unsigne
     return gtNewSimdHWIntrinsicNode(TYP_MASK, NI_Sve_CreateTrueMaskAll, simdBaseJitType, simdSize);
 }
 
+//------------------------------------------------------------------------
+// gtNewSimdAllFalseMaskNode: Create an embedded mask with all bits set to false
+//
+// Arguments:
+//    simdSize        -- the simd size of the nodes being masked
+//
+// Return Value:
+//    The mask
+//
+GenTree* Compiler::gtNewSimdAllFalseMaskNode(unsigned simdSize)
+{
+    return gtNewSimdHWIntrinsicNode(TYP_MASK, NI_Sve_CreateFalseMaskByte, CORINFO_TYPE_BYTE, simdSize);
```

Comment:
I'm not sure on this line.

It should be `switch(type) case byte: NI_Sve_CreateFalseMaskByte; case Int32: NI_Sve_CreateFalseMaskInt32` etc etc to keep to the hwintrinsiclistarm64sve.h interface.

However, regardless of which is used, it'll still produce the same `pfalse` instruction.

Alternatively, add a `NI_Sve_CreateFalseMaskAll` similar to `NI_Sve_CreateTrueMaskAll` which can take any type. But that require support adding to a few additional files.